Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -71,6 +72,8 @@ namespace AzToolsFramework
             using AliasToInstanceMap = AZStd::unordered_map<InstanceAlias, AZStd::unique_ptr<Instance>>;
             using AliasToEntityMap = AZStd::unordered_map<EntityAlias, AZStd::unique_ptr<AZ::Entity>>;
 
+            //! Construtors.
```

Comment:
Doxygen grouping sounds like a cool idea. Just make sure that it appears like you would expect it to on the website documentation.

Reply:
Yeah. I like this idea too. I think it is epecially good for overloaded members.

I checked on documentation website for one existing example. They did end up being grouped together. I put it here for reference.

```cpp
//! Normalizes the vector in-place and returns the previous length.
//! This takes a few more instructions than calling just Normalize().
//! @{
float NormalizeWithLength();
float NormalizeWithLengthEstimate();
//! @}
```

https://www.o3de.org/docs/api/frameworks/azcore/class_a_z_1_1_vector2.html